06_deploy_masters.sh

```diff
@@ -6,7 +6,7 @@ source common.sh
 
 # Note This logic will likely run in a container (on the bootstrap VM)
 # for the final solution, but for now we'll prototype the workflow here
-instack="ocp/master_nodes.json"
+MASTER_NODES_FILE=${MASTER_NODES_FILE:-"ocp/master_nodes.json"}
```

Comment:
ocp/master_nodes.jsonis also mentioned in 05_deploy_bootstrap_vm.sh , should we move this line into common.sh and use it in the other file also?

Reply:
Ah yeah good point, I'll update it